Install_WSL.rst

```diff
@@ -0,0 +1,106 @@
+.. -*- coding: utf-8; indent-tabs-mode:nil; -*-
+
+############################################################
+Spinal Cord Toolbox for Windows : Windows subsyetm for linux
+############################################################
+
+This documentation shows how to install the latest version of SCT. If you want install a different version of SCT please specify the version number you want.
+
+.. contents::
+..
+    1  Windows subsystem for linux 
+    2  environment preparation
+    3  SCT installation 
+    4  FSL installation 
+    5  Usage 
+
+Installation
+************
+
+
+#. Install `Xming <https://sourceforge.net/projects/xming/files/Xming/6.9.0.31/>`_.
```

Comment:
why pointing to a specific Xming release? in 1 year from now, the release will likely have changed

Reply:
I use the same link that we used in the README that's why I took this one. I will change that.